vm/vm.go

```diff
@@ -362,8 +381,8 @@ var (
 	executingProgram1 = []byte("executing program")  // syz-fuzzer, syz-runner output
 	executingProgram2 = []byte("executed programs:") // syz-execprog output
 
-	beforeContext = 1024 << 10
-	afterContext  = 128 << 10
+	beforeContextDefault = 1024 << 10
```

Comment:
pkg/instance sets own default now, so I think this should be 0 now?
otherwise one of these is unused and it's a recipe for future bugs

Reply:
Updated the code so that it treats 0 as a default value. Then removed the duplication of constant as there was no more need to do so.